Tools/Scripts/libraries/resultsdbpy/resultsdbpy/controller/api_routes.py

```diff
@@ -50,7 +51,8 @@ def __init__(self, model, import_name=__name__, auth_decorator=None, bug_tracker
         self.ci_controller = CIController(ci_context=model.ci_context, upload_context=model.upload_context)
         self.archive_controller = ArchiveController(commit_controller=self.commit_controller, archive_context=model.archive_context, upload_context=model.upload_context)
 
-        self.bug_tracker_controller = BugTrackerController(bug_tracker_configs=bug_tracker_configs, commit_context=model.commit_context)
+        self.bug_tracker_configs = [WebKitBugzilla()] if not len(bug_tracker_configs) else bug_tracker_configs
```

Comment:
Can we put a fix-me here? This default is probably going to need to be un-done at some point if another team is interested in using the results database.